src/manifest.json

```diff
@@ -37,6 +37,7 @@
         }
     ],
     "permissions": [
+        "alarms",
```

Comment:
One thing I'm afraid it to cause a warning for users after an update.

Reply:
I don't have time to find all the sources about this, so I'll post them tomorrow. Essentially, `alarms` is one of those odd permisions which must be in the manifest but are granted automatically.

Reply:
I have the following evidence that installing update with the new `alarms` permission does not generate extra warnings:
 - `chrome.management.getPermissionWarningsByManifest` creates the list of user-visible permissions warnings for a given manifest and it returns nothing for a manifest with only `alarms` permission ([Chrome docs](https://developer.chrome.com/docs/extensions/reference/management/#method-getPermissionWarningsByManifest), [MDN](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/management/getPermissionWarningsByManifest))
 - Chrome docs do not list `alarms` among ["Permissions with warnings"](https://developer.chrome.com/docs/extensions/mv2/permission_warnings/#permissions_with_warnings)

Ideally, we would be able to test that Chrome and Firefox do not actually just decide to disable extension anyway upon end-user updating the extension. I'm still researching ways to do this.